### ISSUE TITLE
bug: svg图像和a链接元素缺少可访问性属性

### ISSUE THREAD

**Issue by huangkevin-apr** [umijs/dumi#2310]
### Version

NA

### OS Version

Windows 11

### Node.js Version

22.14.0

### Browser Version

Edge

### Link to minimal reproduction

https://github.com/actions-cool/issues-helper/issues/224

### Steps to reproduce

我在[issues-helper](https://github.com/actions-cool/issues-helper)项目中发现，使用dumi主题渲染出的svg图像具有accessibility violation，这个bug的详情可以参考我在[issues-helper](https://github.com/actions-cool/issues-helper)中提的[ISSUE#224](https://github.com/actions-cool/issues-helper/issues/224)

### What is expected?

每个svg元素都具有可访问性名称，以便于使用屏幕阅读器的盲人用户或低视力用户能够理解每个svg图像的意图以正常访问网页。

### What is actually happening?

当前的svg元素没有提供任何可访问性的属性。
